internal/bundle/bundle.go

```diff
@@ -0,0 +1,12 @@
+package bundle
+
+import (
+	"github.com/rs/zerolog"
+	"github.com/weka/gohomecli/internal/utils"
+)
+
+var logger = utils.GetLogger("bundle")
+
+func SetLogger(log zerolog.Logger) {
+	logger = log.With().Str("component", "bundle").Logger()
```

Comment:
that's what happening under the hood inside utils.GetLogger, what do you need this for?

Reply:
I think it was added because logger has different level, but it's solved. Removed it